public/dorequest.php

```diff
@@ -162,25 +165,30 @@ function DoRequestError(string $error, ?int $status = 200, ?string $code = null)
         }
         break;
 
+    // TODO require authentication?
     case "codenotes2":
         $response['CodeNotes'] = getCodeNotesData($gameID);
         $response['GameID'] = $gameID;
         break;
+
     case "gameid":
         $md5 = request()->input('m') ?? '';
         $response['GameID'] = getGameIDFromMD5($md5);
         break;
 
+    // TODO require authentication?
```

Comment:
This is [exposed through rcheevos](https://github.com/RetroAchievements/rcheevos/wiki/rc_api_init_fetch_games_list_request) without authentication. Clients would have to upgrade, though I'm not sure if any clients besides the DLL use it.